evadb/catalog/services/table_catalog_service.py

```diff
@@ -57,8 +57,20 @@ def insert_entry(
             # populate the table_id for all the columns
             for column in column_list:
                 column.table_id = table_catalog_obj._row_id
-            column_list = self._column_service.insert_entries(column_list)
+            column_list = self._column_service.create_entries(column_list)
 
+            # atomic operation for adding table and its corresponding columns.
+            try:
+                self.session.add_all(column_list)
+                self.session.commit()
+            except Exception as e:
+                self.session.rollback()
+                self.session.delete(table_catalog_obj)
+                self.session.commit()
+                logger.exception(
+                    f"Failed to insert entry into table catalog with exception {str(e)}"
```

Comment:
Is this error supposed to be shown when a table with `_row_id` is column is created?
I still get the same error as earlier:
```
ExecutorError: (sqlite3.IntegrityError) UNIQUE constraint failed: column_catalog.name, column_catalog.table_id                                        
[SQL: INSERT INTO column_catalog (name, type, is_nullable, array_type, array_dimensions, table_id) VALUES (?, ?, ?, ?, ?, ?) RETURNING _row_id]       
[parameters: ('_row_id', 'INTEGER', 1, None, '()', 5)]                                                                                                
(Background on this error at: https://sqlalche.me/e/20/gkpj) 
```

Reply:
I see. I'll raise an error if the user specifies such a column name.